Library/Formula/supersonic.rb

```diff
@@ -0,0 +1,98 @@
+class Supersonic < Formula
+  homepage "https://code.google.com/p/supersonic/"
+  url "https://supersonic.googlecode.com/files/supersonic-0.9.4.tar.gz"
+  sha256 "1592dfd2dc73f0b97298e0d25e51528dc9a94e9e7f4ab525569f63db0442d769"
+
+  depends_on "glog"
+  depends_on "protobuf"
+  depends_on "boost"
+  depends_on "pkg-config" => :build
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}",
+                          "--without-re2"
+    system "make", "clean"
```

Comment:
I thought the answer is positive. I have tried to remove it on my computer and it failed.